tests/expectations/metrics/test_metrics_util.py

```diff
@@ -315,18 +315,24 @@ def test_get_dbms_compatible_metric_domain_column_kwargs_column_not_found(
 
 @pytest.mark.unit
 @pytest.mark.parametrize(
-    "input_column_name, output_column_name, confirm_not_equal_column_name",
     [
-        [
+        "input_column_name",
+        "output_column_name",
+        "confirm_not_equal_column_name",
+    ],
+    [
+        pytest.param(
             "ARTISTS",
             "ARTISTS",
             None,
-        ],
-        [
-            "travel_agents",
-            sqlalchemy.quoted_name(value="travel_agents", quote=True),
+            id="column_does_not_need_to_be_quoted",
+        ),
+        pytest.param(
+            "should_be_quoted",
+            sqlalchemy.quoted_name(value="should_be_quoted", quote=True),
             "TRAVEL_AGENTS",
```

Comment:
How about this?
```suggestion
            "SHOULD_BE_QUOTED",
```

Reply:
@anthonyburdi `TRAVEL_AGENTS` is not to be quoted.  Thanks.